### DISCUSSION TITLE
Kotlin Version

### DISCUSSION THREAD

**Discussion by Martifan** [NightscoutFoundation/xDrip#3046] [Q&A]
hello, maybe someone has a code for Kotlin, or a piece of code, how to connect to the sensor and how to read data from the sensor?
thank you all very much

**Comment by Navid200** [comment]
xDrip is open source so that we can put our efforts together to improve it for the users.

If your intent is to improve xDrip, you can open a PR even if it is in Kotlin.
If you are not breaking xDrip, your PR will be considered.

**Reply by Martifan** [reply]
I can't work in JAVA, so if anyone has a Kotlin code, please share
